docs/networking/deep-dive.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 3
+sidebar_label: Deep Dive
+title: ""
+keywords:
+- Harvester
+- Networking
+- Topology
+---
+# Harvester Network Deep Dive
+
+The network topology below reveals how we implement the Harvester network.
+
+![](/img/v1.1/networking/topology.png)
+
+The diagram contains [the built-in cluster network mgmt](./clusternetwork.md/#built-in-cluster-network) and a [custom cluster network](./clusternetwork.md/#custom-cluster-network) called `oob`.
+
+As shown above, the Harvester network focuses on layer 2. We leverage Linux network devices to construct traffic paths for the communication between VM and VM, VM and host, and VM and external network devices.
+
+The Harvester network is divided into three layers, including:
+
+- kubevirt networking layer
+
+- Harvester networking layer
+
+- external networking layer
+
+## Kubevirt Networking
+
+The basic idea of kubevirt is to put libvirt into the Kubernetes pod. The kubevirt networking is responsible for forwarding the traffic between the pod and the VM inside.
+Please refer to the [kubevirt official document](https://kubevirt.io/2018/KubeVirt-Network-Deep-Dive.html) for more details.
+
+## Harvester Networking
+
+Harvester networking aims to forward the traffic from pods to the host network. It implements the management network, VLAN(tagged) network and untagged network. We temporarily refer to the latter two as **bridge network**, because bridge plays a key role in their implementation.
```

Comment:
```suggestion
Harvester networking is designed to forward traffic from pods to the host network, It implements a management network, VLAN networks and untagged networks. We can refer to the last two networks as bridged networks because bridging plays a vital role in their implementation.
```

Reply:
The word `bridged` looks strange.

Reply:
I prefer to keep the name `bridge network`.